GetStream/stream-chat-android
# [4042] Fallback UI for unsupported attachments in XML

### 🎯 Goal

https://github.com/GetStream/stream-chat-android/issues/4042

### 🛠 Implementation details

Added fallback `UnsupportedAttachmentFactory` for attachments with unsupported types.

### 🎨 UI Changes

| Before | After |
| --- | --- |
| ![photo_2022-10-12 21 28 51](https://user-images.githubusercontent.com/9600921/195420323-3e2c1f16-b030-427a-b3d9-412a29d2d050.jpeg) | ![photo_2022-10-12 21 28 48](https://user-images.githubusercontent.com/9600921/195420331-c3b79128-33c0-4dd6-9c2c-7c27d58a5517.jpeg) |

### 🧪 Testing

1. Open UI Components sample app
2. Login as Dmitrii
3.  Open "unsupported attachments" channel
4. Ensure that unsupported attachments are displayed correctly

### ☑️Contributor Checklist

#### General

#### Code & documentation

### ☑️Reviewer Checklist
